### PR TITLE
Brad edits

### PR DESCRIPTION
**Pull request by bsaund**
Some changes to make the model.fit work without throwing errors.

You can use this as a starting point, but I made changes that you will want to fix properly later

### CODE REVIEW

File: audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import keras
+import tensorflow.keras as keras
```
Comment (bsaund) [inline audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py:1]: keras used to be a separate package, but is now part of tensorflow

File: audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py [hunk 2]
```diff
@@ -218,16 +218,16 @@ def preprocess_dataset(files):
...
-    layers.Conv1D(32, 3, activation='relu'),    # changed all these to 1D
-    layers.Conv1D(64, 3, activation='relu'),
-    layers.MaxPooling1D(),
-    layers.Dropout(0.25),
-    layers.Flatten(),
-    layers.Dense(128, activation='relu'),
-    layers.Dropout(0.5),
+    layers.Input(shape=[None, 1000]),
+    # preprocessing.Resizing(32, 32),
+    # norm_layer,
```
Comment (bsaund) [inline audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py:223]: I just got rid of these to remove potential places for the tensorflow errors to occur. Add them back in when you are ready

File: audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py [hunk 3]
```diff
@@ -218,16 +218,16 @@ def preprocess_dataset(files):
...
-    preprocessing.Resizing(32, 32),
-    norm_layer,
-    layers.Conv1D(32, 3, activation='relu'),    # changed all these to 1D
-    layers.Conv1D(64, 3, activation='relu'),
-    layers.MaxPooling1D(),
-    layers.Dropout(0.25),
-    layers.Flatten(),
-    layers.Dense(128, activation='relu'),
-    layers.Dropout(0.5),
+    layers.Input(shape=[None, 1000]),
```
Comment (bsaund) [inline audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py:221]: I have hardcoded the length to 1000. You'll want to change that eventually

File: audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py [hunk 4]
```diff
@@ -242,21 +242,33 @@ def preprocess_dataset(files):
 # compile, aka define the optimizer and loss
 model.compile(
     optimizer=tf.keras.optimizers.Adam(),
-    loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+    loss=tf.keras.losses.CategoricalCrossentropy(from_logits=True),
```
Comment (bsaund) [inline audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py:245]: SparseCategoricalCrossentropy vs CategoricalCrossentropy just indicated if you expect the data to be one-hot encoded

File: audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py [hunk 5]
```diff
@@ -242,21 +242,33 @@ def preprocess_dataset(files):
...
 )
 
+
+labs_categories = {"up": 0, "left": 1, "right": 2, "down": 3, "no": 4, "yes": 5, "go": 6, "stop": 7}
+def categorize(command: str):
+    v = np.zeros([num_labels])
+    v[labs_categories[command]] = 1.0
+    return v
+
+labs_vec = np.array([categorize(s) for s in labs])
```
Comment (bsaund) [inline audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py:256]: I had to convert the string labels into numbers for a loss function to make sense. I chose the one-hot encoding.

File: audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py [hunk 6]
```diff
@@ -242,21 +242,33 @@ def preprocess_dataset(files):
...
+truncated_wavs = np.array([w[0:1000] for w in wavs])
```
Comment (bsaund) [inline audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py:257]: To use `model.fit` your data must be in specific formats (see tensorflow documentation). One format is as a single numpy array. However, to convert `wavs` into a numpy array (of floats, as opposed to complicated types), the length of each `wav` must be the same. I just truncated each `wav` to 1000 to get things working. Obviously you'll want to fix this. You cant just use a single numpy array though, because you want variable sized `wav` inputs. This is where my `model_runner` and `my_keras_model` allow you to deal with these details
